draft-ietf-quic-recovery.md

```diff
@@ -591,15 +592,10 @@ client, it is the client's responsibility to send packets to unblock the server
 until it is certain that the server has finished its address validation
 (see Section 8 of {{QUIC-TRANSPORT}}).  That is, the client MUST set the
 probe timer if the client has not received an acknowledgement for one of its
-Handshake or 1-RTT packets, and has not received a HANDSHAKE_DONE frame.
-If Handshake keys are available to the client, it MUST send a Handshake
-packet, and otherwise it MUST send an Initial packet in a UDP datagram of
-at least 1200 bytes.
-
-A client could have received and acknowledged a Handshake packet, causing it to
-discard state for the Initial packet number space, but not sent any
-ack-eliciting Handshake packets.  In this case, the PTO timer is armed from the
-time that the Initial packet number space is discarded.
+Handshake or 1-RTT packets, and has not received a HANDSHAKE_DONE frame,
+even if there are no packets in flight. If Handshake keys are available to the
+client, it MUST send a Handshake packet, and otherwise it MUST send an Initial
+packet in a UDP datagram of at least 1200 bytes.
```

Comment:
```suggestion
even if there are no packets in flight. When the PTO fires, the client MUST
send a Handshake packet if it has Handshake keys, otherwise it MUST
send an Initial packet in a UDP datagram of at least 1200 bytes.
```